diff --git a/glue-codes/python/pyproject.toml b/glue-codes/python/pyproject.toml
index 133d56967..6d4633313 100644
--- a/glue-codes/python/pyproject.toml
+++ b/glue-codes/python/pyproject.toml
@@ -22,20 +22,21 @@ classifiers = [
     "Programming Language :: Python :: 3.11",
     "Programming Language :: Python :: 3.12",
     "Programming Language :: Python :: 3.13",
+    "Programming Language :: Python :: 3.14",
     "Programming Language :: Python :: Implementation :: CPython",
     "Programming Language :: Python :: Implementation :: PyPy"
 ]
 dependencies = [
-    "numpy~=2.0",
-    "matplotlib~=3.0",
+    "numpy",
+    "matplotlib",
 ]
 
 [project.optional-dependencies]
 windio = [
-    "windio~=1.0",
+    "windio",
 ]
 develop = [
-    "pytest~=8.0",
+    "pytest",
 ]
 
 [tool.setuptools.packages.find]
diff --git a/openfast_io/openfast_io/turbsim_util.py b/openfast_io/openfast_io/turbsim_util.py
index 73e1c0cc3..a47dd3ab5 100644
--- a/openfast_io/openfast_io/turbsim_util.py
+++ b/openfast_io/openfast_io/turbsim_util.py
@@ -90,6 +90,8 @@ class TurbsimReader(object):
         self.CTLz = inpf.readline().split()[0]
         self.CTStartTime = inpf.readline().split()[0]
 
+        inpf.close()
+
 
 class TurbsimWriter(object):
 
diff --git a/openfast_io/pyproject.toml b/openfast_io/pyproject.toml
index b942dade9..c356f47b5 100644
--- a/openfast_io/pyproject.toml
+++ b/openfast_io/pyproject.toml
@@ -42,14 +42,18 @@ classifiers = [  # Optional
   "Programming Language :: Python :: 3.10",
   "Programming Language :: Python :: 3.11",
   "Programming Language :: Python :: 3.12",
+  "Programming Language :: Python :: 3.13",
+  "Programming Language :: Python :: 3.14",
   "Programming Language :: Python :: 3 :: Only",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python",
 ]
 
 dependencies = [
-    "numpy>1.0",
-    "pandas>2.0",
-    "ruamel_yaml>0.18",
-    "deepdiff>8.0",
+    "numpy",
+    "pandas",
+    "ruamel_yaml",
+    "deepdiff",
     ]
     
 [project.urls]
@@ -62,9 +66,9 @@ Issues = "https://github.com/OpenFAST/openfast/issues"
 
 [project.optional-dependencies]
 
-rosco = ["rosco>2.9.2"]
-xlrd = ["xlrd>2"]
-all = ["rosco>2.9.2", "xlrd>2"]
+rosco = ["rosco"]
+xlrd = ["xlrd"]
+all = ["rosco", "xlrd"]
 
 [tool.hatch.version]
 source = "vcs"
